Formula/manifest-tool.rb

```diff
@@ -0,0 +1,35 @@
+class ManifestTool < Formula
+  desc "Command-line tool to create and query container image manifest list/indexes"
+  homepage "https://github.com/estesp/manifest-tool/"
+  url "https://github.com/estesp/manifest-tool/archive/refs/tags/v2.0.3.tar.gz"
+  sha256 "53bba7cc7465c8347c70e36467848bd385eba2fb51d7aceb269c316382a065a7"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    package = "busybox:latest"
+    stdout, stderr, = Open3.capture3(
+      bin/"manifest-tool", "inspect",
+      package
+    )
```

Comment:
Why not `shell_output`?

Reply:
afaict `shell_output` has a single output. `manifest-tool` sends its error about `429` to `stderr` and its normal output to `stdout`. It's slightly more technically correct to only look for the 429 in `stderr` and the output stuff in `stdout`.

I'm not remotely wed to this and will happily change if requested. riff.rb and xx.rb use it. I think I took my snippet from the latter since it both matches the general style, and I didn't consider the result code.

Reply:
It's much easier to redirect the output of stderr and just use `shell_output` though

Reply:
It's not easy to tell `shell_output` to expect different exit codes depending on the output of the command, though.